clang/test/Driver/sycl-offload.c

```diff
@@ -87,6 +87,42 @@
 
 /// ###########################################################################
 
+/// Check error for -fsycl-targets with bad triple
+// RUN:   %clang -### -fsycl-targets=spir64_bad-unknown-unknown-sycldevice -fsycl  %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHK-SYCL-FPGA-BAD-TRIPLE %s
+// RUN:   %clang_cl -### -fsycl-targets=spir64_bad-unknown-unknown-sycldevice -fsycl  %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHK-SYCL-FPGA-BAD-TRIPLE %s
+// CHK-SYCL-FPGA-BAD-TRIPLE: error: SYCL target is invalid: 'spir64_bad-unknown-unknown-sycldevice'
+
```

Comment:
It's true that `-fsycl-link-targets` gradually decreases in usefullness, but I think we should still cover it.

Reply:
Added some -fsycl-link-targets triple tests